packages/lisk-transactions/src/transaction_types.ts

```diff
@@ -29,9 +54,24 @@ export interface BaseTransaction {
 	readonly type: number;
 }
 
+export interface VerifyReturn {
+	readonly errors?: ReadonlyArray<TransactionError>;
+	readonly verified: boolean;
+}
+
+export interface ValidateReturn {
+	readonly errors?: ReadonlyArray<TransactionError>;
+	readonly validated: boolean;
+}
+
+export interface StateReturn {
+	readonly recipient?: Account;
+	readonly sender: Account;
+}
+
 type Partial<T> = { [P in keyof T]?: T[P] };
```

Comment:
it's not directly related to this PR, but we can remove this.
`Partial` is actually defined in typescript itself